sbi/neural_nets/density_estimators/flow.py

```diff
@@ -13,10 +15,6 @@ class NFlowsFlow(DensityEstimator):
     wrap them and add the .loss() method.
     """
 
-    def __init__(self, net: flows.Flow):
-
-        super().__init__(net)
-
     def log_prob(self, input: Tensor, condition: Tensor) -> Tensor:
         r"""Return the batched log probabilities of the inputs given the conditions.
```

Comment:
add a note about the broadcasting behavior here in the docstring?

Reply:
I added Note to the base class `DensityEstimator`, as this is what everyone should satisfy.

I will also copy it over here.